sklearn/metrics/tests/test_classification.py

```diff
@@ -1599,6 +1600,32 @@ def test_log_loss():
     loss = log_loss(y_true, y_score2, labels=[1, 2, 3])
     assert_almost_equal(loss, 1.0630345, decimal=6)
 
+    # test multi-label #1
```

Comment:
I would like to see the multilabel tests structured as invariants with respect to the multiclass definition. In particular, we should be asserting that for some multilabel y_true and y_proba, the following holds:
```py
per_label_loss = []
for i in range(y_true.shape[1]):
    y_proba_i = np.vstack(y_proba[:, i], 1 - y_proba[:, i]).T
    per_label_loss.append(log_loss(y_true[:, i], y_proba_i))

assert log_loss(y_true, y_proba) = np.mean(per_label_loss)
```

Reply:
What I mean is that I don't want to see magic numbers here, like `0.8082688`. the reader has no reason to believe that is correct. But invariance relative to the binary case is meaningful

Reply:
I see. makes sense.